internal/pkg/cli/svc_init.go

```diff
@@ -126,7 +126,8 @@ type initSvcOpts struct {
 	topics       []manifest.TopicSubscription
 
 	// Cache variables
-	df dockerfileParser
+	wsAppName string
```

Comment:
Q: Are you making this a cached variable rather than calling `tryReadingAppName()` within `Ask()` for ease of testing? Or are there other benefits?

Reply:
it's because `appName` is now no longer a required flag so we put it within `Validate`. and cached variable is for ease of testing.